yaksh/templates/yaksh/user_data.html

```diff
@@ -83,7 +84,7 @@
             {%endif%}
         </div>
     </div>
-    {% if question.type == "mcq" or question.type == "mcc" %}
+    {% if question.type != "code" %}
```

Comment:
No, this doesn't break. Because in django, if it doesnt find a django variable while rendering, it will not raise an exception, it will just render with the variable value as none.